packages/project-disaster-trail/src/components/Game/TaskScreen/ChooseScreen.js

```diff
@@ -159,12 +133,14 @@ ChooseScreen.propTypes = {
   weightedTasks: PropTypes.arrayOf(PropTypes.shape({})),
   addNextTask: PropTypes.func,
   activeEnvironment: PropTypes.string,
-  tasksForEnvironment: PropTypes.shape({})
+  tasksForEnvironment: PropTypes.shape({}),
+  votingComplete: PropTypes.bool
+  // startTick: PropTypes.func,
+  // stopTick: PropTypes.func,
```

Comment:
Can remove this commented out code it looks like

Reply:
🦅 👁